sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -100,8 +100,12 @@ es3fLifetimeTests.ScaleProgram.prototype.draw = function(vao, scale, tf, dst) {
 es3fLifetimeTests.ScaleProgram.prototype.setPos = function(buffer, vao) {
     gl.bindBuffer(gl.ARRAY_BUFFER, buffer);
     gl.bindVertexArray(vao);
-    if (buffer)
+    if (buffer) {
         gl.vertexAttribPointer(this.m_posLoc, NUM_COMPONENTS, gl.FLOAT, false, 0, 0);
```

Comment:
Delete the buffer will remove unbind the buffer from vertex attribute array. Otherwise detach (called from glsLifetimeTests.AttachmentTest.testDeletedNames) does nothing. Or are there any other methods to detach array buffer?